test/screenshot/spec/mdc-side-sheet/fixture.scss

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "../../../../packages/mdc-list/mixins";
+@import "../../../../packages/mdc-side-sheet/mixins";
+@import "../../../../packages/mdc-side-sheet/variables";
+@import "../../../../packages/mdc-theme/color-palette";
+@import "../../../../packages/mdc-top-app-bar/mdc-top-app-bar";
+@import "../../../../packages/mdc-typography/mdc-typography";
+@import "../../../../packages/mdc-typography/mixins";
+
+$custom-side-sheet-color: $material-color-red-600;
+
+.test-viewport--side-sheet {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.test-side-sheet-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.test-side-sheet-app-content {
+  flex: auto;
+}
+
+.test-side-sheet-top-app-bar {
+  position: absolute;
+}
+
+.test-side-sheet-main-content {
+  display: block; // Needed for IE 11
+  box-sizing: border-box;
+  height: 100%;
+  padding-right: 16px;
+  padding-left: 16px;
+  overflow: auto;
+}
+
+.test-side-sheet > :first-child,
```

Comment:
Are these to avoid UA differences?

Reply:
No, they're to avoid giving the false impression that `__content` has padding.
I added a comment here explaining that.

The default margins on `<p>` elements could easily be mistaken for container padding, and I want to make it obvious that `__content` does not have any padding.

Another option would be to remove the `<p>` elements and use `<br>` tags for spacing.